code/modules/bitrunning/components/avatar_gear.dm

```diff
@@ -0,0 +1,67 @@
+
+/// Tracks the highest up human carrying this item, and loads bitrunning gear onto their avatar when they enter a virtual domain
+/datum/component/loads_avatar_gear
+	/// The callback called when COMSIG_BITRUNNER_STOCKING_GEAR is sent to our host human
+	var/datum/callback/load_callback
+	/// The human we are currently carried by
+	var/mob/living/carbon/human/tracked_human
```

Comment:
is this cleared on destroy?

Reply:
nnnnooooo
thanks I converted it to a weakref